app/controllers/incomeSources/add/AddIncomeSourceStartDateCheckController.scala

```diff
@@ -199,6 +201,10 @@ class AddIncomeSourceStartDateCheckController @Inject()(authenticate: Authentica
           )
       case (Some(form.responseYes), _) =>
         Redirect(successCall)
+      case (Some(_), _) => Logger("application").error(s"[AddIncomeSourceStartDateCheckController][handleValidForm] - Unexpected response")
```

Comment:
can we change log messages below so they would not be identical and we need details if this is an Agent or Individual